packages/appstore/src/constants/trading-hub-content.ts

```diff
@@ -0,0 +1,89 @@
+import { localize } from '@deriv/translations';
+import React from 'react';
+
+type TTradingHubContents = {
+    step1: {
```

Comment:
Is it possible to extract the step type and used it here?
Like:

```
type TStep = {
        component: React.ReactNode;
        footer_header: string;
        footer_text: string;
        next_content?: string;
        has_next_content: boolean;
}
type TTradingHubContents = {
        step1: TStep;
}

```

Reply:
This type (`TStep`) is also used in `onboarding.tsx` file. [here](https://github.com/binary-com/deriv-app/pull/6309/files#diff-4e423c0b7a087a6df6c3a3bd78ec8734cb52cd896c7f62a78a0dc7684675732bR6).